vivo-project/Vitro
# [Vivo 1918] i18n: Reload firsttime files on start-up if changed

**[JIRA Issue](https://jira.lyrasis.org/browse/VIVO-1918)**

# What does this pull request do?
This PR adds the functionality requested in the JIRA-Ticket
On the firsttime Start-Up a backup of the firsttime files is saved to a new model.
On every Start-Up these backup will be compared to the current state of the firsttime files. If the files have changed, try to apply
these changes to the user model. So if you activate a new language or change stuff in some firsttime files, you will no longer need to do a firsttime Start-Up of your system, with a normal restart the changes will be applied. 

# How should this be tested?
You need a fresh system (firsttime Start-Up) - for that, you can delete the tdb* directories in vivo_home (if you do not want to set up a whole new system)
Start the system so the new backup models will be created. After this, change some firsttime files and restart the system.
The changes should now be available without a new firsttime Start-Up. 

good examples:
- vivo_home_dir/rdf/i18n/en_US/tbox/firsttime/vitroAnnotations_en_US.n3
- vivo_home_dir/rdf/i18n/en_US/display/firsttime/menu_en_US.nt
